src/wallet/api/wallet2_api.h

```diff
@@ -927,6 +927,13 @@ struct Wallet
      */
     virtual bool importOutputs(const std::string &filename) = 0;
 
+    /*!
+     * \brief scanTransactions - scan a list of transaction ids, this operation may reveal the txids to the remote node and affect your privacy
+     * \param txids            - list of transaction ids
+     * \return                 - true on success
```

Comment:
"True on success, meaning, that all transactions were valid"?

Reply:
Success meaning all the transactions got successfully scanned.